### PR TITLE
Add status return to cleanup service

### PR DESCRIPTION
**Pull request by seanpang-vmware**
Use Status which contains errorcodes and error message in cleanup service.

### CODE REVIEW

File: pkg/clean/clean.go [hunk 1]
```diff
@@ -47,27 +52,22 @@ func Clean(cf *config.NSXOperatorConfig, client *http.Client) error {
...
 		}
 	}
 	log.Info("cleanup NSX resources successfully")
-	return nil
+	return OK, nil
 }
 
 // InitializeCleanupService initializes all the CR services
-func InitializeCleanupService(cf *config.NSXOperatorConfig, client *http.Client) (*CleanupService, error) {
+func InitializeCleanupService(cf *config.NSXOperatorConfig, client *http.Client, nsxClient *nsx.Client) (*CleanupService, error) {
```
Comment (zhengxiexie) [inline pkg/clean/clean.go:68]: client *http.Client should be removed if nsxClient is here.

**Review by zhengxiexie — APPROVED** [review]
LGTM

**Review by dantingl — APPROVED** [review]
LGTM

**Pull request merged**